4/test/excercise4.spec.js

```diff
@@ -0,0 +1,123 @@
+const { gql } = require('apollo-server')
+const server = require('../server.js')
+const {createTestClient} = require('apollo-server-testing');
+//const jwt = require('jsonwebtoken');
+
+const { query, mutate } = createTestClient(server);
+
+const mutateDeleteAllTodos = gql`mutation deleteAllTodos { deleteAllTodos }`;
+const mutateDeleteAllAssignees = gql`mutation deleteAllAssignees { deleteAllAssignees }`;
+
+const mutateCreateAssignee = gql`mutation createAssignee($id: ID, $name: String!) {
+    createAssignee(id: $id, name: $name) { id, name } 
+}`;
+
+const mutateCreateTodo = gql`mutation createTodo($id: ID, $message: String!, $state: State) {
+    createTodo(id: $id, message: $message, state: $state) { id, message } 
+}`;
+
+const mutateCreateRelationship = gql`mutation createRelationship($id: ID!, $name: String!) {
+    createRelationship(id: $id, name: $name) { id } 
+}`;
+
+describe('Creation of assignees and todo items', () => {
+
+    const assignee = { id: '100', name: 'SomeoneElse'};
+    const todo = { id: '200', message: 'Just another new task', state: 'IN_PROGRESS'};  
+
+    beforeAll(async () => {
+        await mutate({mutation: mutateDeleteAllTodos});
+        await mutate({mutation: mutateDeleteAllAssignees});
+    })
+
+    afterAll(async () => {
+        await mutate({mutation: mutateDeleteAllTodos});
+        await mutate({mutation: mutateDeleteAllAssignees});
+    })
+
+    it('Add assignee to db', async () => {
```

Comment:
Write an third person English sentence in simple present.
```suggestion
    it('adds assignee to db', async () => {
```